metadata-ingestion/README.md

```diff
@@ -483,13 +483,16 @@ Pull metadata from DBT output files:
 - [dbt catalog file](https://docs.getdbt.com/reference/artifacts/catalog-json)
   - This file contains schema data.
   - DBT does not record schema data for Ephemeral models, as such datahub will show Ephemeral models in the lineage, however there will be no associated schema for Ephemeral models
+- target_platform: Optional
+  - In this case we use DBT metadata to enrich an existing data platform with DBT sourced lineage and metadata.
 
 ```yml
 source:
   type: "dbt"
   config:
     manifest_path: "./path/dbt/manifest_file.json"
     catalog_path: "./path/dbt/catalog_file.json"
+    target_platform: "dbt" # optional eg postgresql, snowflake etc.
```

Comment:
It should be `postgres` instead of `postgresql`. Also, can we add a link to https://github.com/linkedin/datahub/blob/master/gms/impl/src/main/resources/DataPlatformInfo.json.